src/modules/luksbootkeyfile/main.py

```diff
@@ -53,7 +54,8 @@ def run():
 
     if not luks_root_passphrase:
         return ("Encrypted rootfs setup error",
-                "Rootfs partition {!s} is LUKS but no passphrase found.".format(luks_root_device))
+                "Rootfs partition {!s}".format(luks_root_device),
+                "is LUKS but no passphrase found.")
 
```

Comment:
, instead of +, but this is also hard on translations. I'm going to reject this PR, because I've fixed it (based on this PR) in master already.